validator/beacon/service.go

```diff
@@ -289,3 +298,10 @@ func (s *Service) CurrentCycleStartSlot() uint64 {
 	cycleNum := math.Floor(float64(currentSlot) / float64(params.DefaultConfig().CycleLength))
 	return uint64(cycleNum) * params.DefaultConfig().CycleLength
 }
+
+// CurrentDemoCycleStartSlot returns the slot at which the current cycle started for demo config.
+func (s *Service) CurrentDemoCycleStartSlot() uint64 {
```

Comment:
This method looks identical to `CurrentCycleStartSlot`. Am I missing something here?

Reply:
One is using the demo config of CycleLength vs. the regular CycleLength - we can just pass that in as a parameter and keep it as one method